### PR TITLE
cluster helpers

### LINKED ISSUE #189: Demo/proof-of-concept for helper notebooks
Convert nwm streamflow tutorials to use cluster and aws config 'helper' functions.

### PR DESCRIPTION
**Pull request by gzt5142**
closes #189

### CODE REVIEW

**Comment by gzt5142** [comment]
helper notebooks created... now on to demo how they are used/imported.

**Comment by gzt5142** [comment]
OK... I think I have these working as I described when we talked about it last week. 

I worked the DScore and StandardSuite streamflow tutorials to use the helpers stored in environment_set_up.

**Review by amsnyder — CHANGES_REQUESTED** [review]
I think we should have the helpers all in one folder (maybe with a README if it isn't too much work to give a short description of the environments for each helper) and then link to that folder in the notebook so that users can see what their environment options are - otherwise, this looks good!

Also found two typos - I would have committed them, but I couldn't suggest these changes in the UI for the file `model_evaluation/tutorials/nwm_streamflow/01_Data_Prep.ipynb` because the diff was too big to load in my browser. If you make the change to the folder structure on your own branch, can you fix the typos too?

**Comment by gzt5142** [comment]
> I think we should have the helpers all in one folder (maybe with a README

I'm confused by this.  They are in the same folder: All the new helpers are in `./environment_set_up`, _as directed_. This is where I was told to put them when we talked about this on our team call. If this is no longer the right spot, then we should talk about where that might be. 

As to a README... I for-sure can expand the existing readme in this folder to describe what these are. Or write one in the new location, once that is determined.

**Comment by amsnyder** [comment]
> > I think we should have the helpers all in one folder (maybe with a README
> 
> I'm confused by this. They are in the same folder: All the new helpers are in `./environment_set_up`, _as directed_. This is where I was told to put them when we talked about this on our team call. If this is no longer the right spot, then we should talk about where that might be.
> 
> As to a README... I for-sure can expand the existing readme in this folder to describe what these are. Or write one in the new location, once that is determined.

Right, they should be in `environment_set_up`! My want is for a user to have a quick way to see what helpers are available to them. I was suggesting we group the helper notebooks into a new sub-directory `environment_set_up\helpers` and add a readme to describe each option. If it's a pain to move them, we could just update the README in `environment_set_up` to describe these helper functions in general and then list the options. Then we might want to link to this README in the notebook so that someone using the notebook could quickly figure out how to adapt if they move to a new computing environment.

**Comment by gzt5142** [comment]
Short blurb in the environment setup readme.

**Comment by amsnyder** [comment]
@gzt5142 can I change the name of `model_evaluation/tutorials/nwm_streamflow/02_Analysis_NWM.ipynb` to `model_evaluation/tutorials/nwm_streamflow/02_Analysis_StdSuite.ipynb`? I will also update the fp for where you read in the helper there since we updated that name.

File: environment_set_up/README.md [hunk 1]
```diff
@@ -18,5 +18,14 @@ options, in increasing order of complexity and flexibility:
    server from a command shell on the HPC. Requires familiarity with the HPC command line, file
    editing, etc.
 
-If your workflow makes use of parallelism using Dask on HPC hosts, read [this](./tallgrass_dask-jobqueue.md)
```
Comment (amsnyder) [inline environment_set_up/README.md:21]: Should we also delete the ./tallgrass_dask-jobqueue.md file? Do we want to link to this notebook here for better understanding instead? https://github.com/hytest-org/hytest/blob/main/essential_reading/Parallel_Dask.ipynb

**Review by amsnyder — APPROVED** [review]

**Pull request merged**